cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/AbstractDataprocProvisioner.java

```diff
@@ -60,7 +60,7 @@ public abstract class AbstractDataprocProvisioner implements Provisioner {
 
   // The property name for the GCS bucket used by the runtime job manager for launching jobs via the job API
   // It can be overridden by profile runtime arguments (system.profile.properties.bucket)
```

Comment:
nit - `It can be overridden by profile runtime arguments (system.profile.properties.gcsBucket)`

Reply:
Done